app/models/edition/custom_lead_image.rb

```diff
@@ -23,8 +25,12 @@ def non_lead_images
 
 private
 
-  def oldest_image
-    images.order(:created_at, :id).first
+  def non_svg_images
+    images.reject { |image| image.content_type == SVG_CONTENT_TYPE }
+  end
+
+  def oldest_non_svg_image
+    non_svg_images.min_by { |image| [image.created_at, image.id] }
```

Comment:
Similarly, it might be interesting to consider moving some of these types of relationship query methods into [active record extensions](https://guides.rubyonrails.org/association_basics.html#association-extensions). That's probably out of the scope of this PR though as I don't believe I've seen AR extensions used at all in Whitehall